pkg/agent/openflow/pod_connectivity.go

```diff
@@ -141,13 +143,15 @@ func (f *featurePodConnectivity) initFlows() []binding.Flow {
 				flows = append(flows, f.arpSpoofGuardFlow(f.nodeConfig.NodeIPv4Addr.IP, gatewayMAC, f.gatewayPort))
 				flows = append(flows, f.hostBridgeUplinkVLANFlows()...)
 			}
-			if runtime.IsWindowsPlatform() || f.connectUplinkToBridge {
-				// This installs the flows between bridge local port and uplink port to support host networking.
-				flows = append(flows, f.hostBridgeUplinkFlows()...)
-			}
+			podCIDRMap[binding.ProtocolIP] = *f.nodeConfig.PodIPv4CIDR
 		}
 	}
+	if runtime.IsWindowsPlatform() || f.connectUplinkToBridge {
+		// This installs the flows between bridge local port and uplink port to support host networking.
+		flows = append(flows, f.hostBridgeUplinkFlows()...)
+	}
 	if f.connectUplinkToBridge {
+		flows = append(flows, f.hostBridgeUplinkVLANFlows()...)
```

Comment:
The flow is added in line 144 too.

Reply:
Thanks. Removed line 144.
This mistake is related to a rebase conflict.